mne_bids/tests/test_path.py

```diff
@@ -19,7 +19,7 @@
                       BIDSPath, write_raw_bids, read_raw_bids,
                       write_meg_calibration, write_meg_crosstalk)
 from mne_bids.path import (_parse_ext, get_entities_from_fname,
-                           _find_best_candidates, _find_matching_sidecar,
+                           _find_best_candidates,
```

Comment:
I changed the private `_find_matching_sidecar` calls in this file to use the public interface. There are other places the private function is used like `test_write.py` but it doesn't seem necessary to change them all here (code churn / blame obfuscation with little benefit)